src/kani-compiler/rustc_codegen_kani/src/utils/utils.rs

```diff
@@ -48,6 +50,16 @@ impl<'tcx> GotocCtx<'tcx> {
         Expr::statement_expression(body, t).with_location(loc)
     }
 
+    /// Generates an expression `((dst as usize) % align_of(T) == 0`
+    /// to determine if `dst` is aligned.
+    pub fn is_aligned(&mut self, typ: &'tcx TyS<'_>, dst: Expr) -> Expr {
```

Comment:
@tautschnig  Will CBMC do the right thing and lower this to a bit operation? Or should we do that ourselves?